editors/TODO

```diff
@@ -1,22 +1,13 @@
 Emacs:
 
-- add shortcut C-c C-r for eglot-reconnect
-
-- When doing C-c C-c on a comment, expand the green until this comment (and not until the following command).
-
 - Do not change the green zone and the goals when editing or adding a comment in the green zone.
 
-- Add buttons for forward/backward navigation.
```

Comment:
I was thinking here about new buttons for C-c C-f/b.

Reply:
Do you mean new icons?

Reply:
Right, new icons. Aren't the current arrow icons for C-c C-n/p?

Reply:
There are also buttons for C-c C-n/p. Try expanding the window.

Reply:
Yes. I took the current icons from `/usr/share/emacs/27.1/etc/images`. It is also possible to add custom icons.

Reply:
In this case, you should check the license/rights and add a editors/emacs/THANKS file.

Reply:
done